content/Rust-1.90.0.md

```diff
@@ -0,0 +1,107 @@
++++
+path = "2025/09/18/Rust-1.90.0"
+title = "Announcing Rust 1.90.0"
+authors = ["The Rust Release Team"]
+aliases = ["releases/1.90.0"]
+
+[extra]
+release = true
++++
+
+The Rust team is happy to announce a new version of Rust, 1.90.0. Rust is a programming language empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via `rustup`, you can get 1.90.0 with:
+
+```console
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`](https://www.rust-lang.org/install.html) from the appropriate page on our website, and check out the [detailed release notes for 1.90.0](https://doc.rust-lang.org/stable/releases.html#version-1900-2025-09-18).
+
+If you'd like to help us out by testing future releases, you might consider updating locally to use the beta channel (`rustup default beta`) or the nightly channel (`rustup default nightly`). Please [report](https://github.com/rust-lang/rust/issues/new/choose) any bugs you might come across!
+
+## What's in 1.90.0 stable
+
+# LLD is now the default linker on `x86_64-unknown-linux-gnu`
+
+The `x86_64-unknown-linux-gnu` target will now use the LLD linker for linking Rust crates by default. This should result in improved linking performance vs the default Linux linker (BFD), particularly for large binaries, binaries with a lot of debug information, and for incremental rebuilds.
+
+In the vast majority of cases, LLD should be backwards compatible with BFD, and you should not see any difference other than reduced compilation time. However, if you do run into any new linker issues, you can always opt out using the `-C linker-features=-lld` compiler flag. Either by adding it to the usual `RUSTFLAGS` environment variable, or to a project's [`.cargo/config.toml`](https://doc.rust-lang.org/cargo/reference/config.html) configuration file,
+like so:
+
+```toml
+[target.x86_64-unknown-linux-gnu]
+rustflags = ["-Clinker-features=-lld"]
+```
+
+If you encounter any issues with the LLD linker, please [let us know](https://github.com/rust-lang/rust/issues/new/choose). You can read more about the switch to LLD, some benchmark numbers and the opt out mechanism [here](https://blog.rust-lang.org/2025/09/01/rust-lld-on-1.90.0-stable/).
+
+### Cargo adds native support for workspace publishing
+
+`cargo publish --workspace` is now supported, automatically publishing all of
+the crates in a workspace in the right order (following any dependencies
+between them). This has long been possible with external tooling or manual
+ordering of individual publishes, but this brings the functionality into Cargo
+itself.
```

Comment:
Should we call out the caveat that the publish is not atomic and network or server side errors can leave you in a partially published state?

This is true for existing workarounds but people might have a different expectation with it built in.